### PR TITLE
Documenting read-only rootfs for issue 17927

### PR DESCRIPTION
**Pull request by JayBeale**
Issue 17927 (https://github.com/kubernetes/website/issues/17927) asks for a change to this task-focused doc on security-context. It notes that this doesn't mention read-only root filesystem, while it's documented in the PSP page. This adds a bullet for read-only rootfs, clarifies that the bulleted list isn't comprehensive, and tells someone reading the list where to find more. The link is copied from the existing What's Next link list.

### CODE REVIEW

**Comment by kbhawkey** [comment]
Hello @JayBeale .
Thanks for opening this pull request.
Page preview:
https://deploy-preview-20793--kubernetes-io-master-staging.netlify.app/docs/tasks/configure-pod-container/security-context/

File: content/en/docs/tasks/configure-pod-container/security-context.md [hunk 1]
```diff
@@ -28,6 +28,12 @@ a Pod or Container. Security context settings include:
 
 * AllowPrivilegeEscalation: Controls whether a process can gain more privileges than its parent process. This bool directly controls whether the [`no_new_privs`](https://www.kernel.org/doc/Documentation/prctl/no_new_privs.txt) flag gets set on the container process. AllowPrivilegeEscalation is true always when the container is: 1) run as Privileged OR 2) has `CAP_SYS_ADMIN`.
 
+* readOnlyRootFilesystem: Mounts the container's root filesystem as read-only.
+
+The above bullets are not a complete set of security context settings -- please see
```
Comment (kbhawkey) [inline content/en/docs/tasks/configure-pod-container/security-context.md:33]: @JayBeale , What do you think about updating lines 33-35:

For a complete list of security context settings, see the Kubernetes [SecurityContext](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#securitycontext-v1-core) API.

Comment (JayBeale) [reply content/en/docs/tasks/configure-pod-container/security-context.md:33]: That sounds perfect @kbhawkey!

Comment (kbhawkey) [inline content/en/docs/tasks/configure-pod-container/security-context.md:33]: nit: I would reword `The above bullets`. Instead:
For the complete set [or list] of security context settings, see 
nit: Our style guide suggests not to use the word `please`.
